StreamChatUITestsAppUITests/Robots/BasicAssertions.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+
+extension Robot {
```

Comment:
Would be nice to document the methods here, since I guess we will be using them a lot.

Reply:
@martinmitrevski Thanks for the comment. We will add more comments to this section :)

Reply:
@martinmitrevski, covered the Robots with the docs, thanks! 🙂